doc/Learn/Maui/ThirdParty-MauiCommunityToolkit.md

```diff
@@ -9,22 +9,66 @@ The controls from MauiCommunityToolkit can be used in an Uno Platform applicatio
 
 An existing sample app that showcases the controls is available [here](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MauiEmbedding/MauiCommunityToolkitApp).
 
+> [!NOTE]
+> For this sample you don't need to have a license, because it's just a demo and for development purposes.
```

Comment:
@Kunal22shah No need for a license for .Net Maui Community Toolkit, you can remove this line.